src/shared/bootspec.c

```diff
@@ -275,20 +275,18 @@ static int boot_entry_compare(const BootEntry *a, const BootEntry *b) {
 }
 
 static int boot_entries_find(
+                BootConfig *config,
                 const char *root,
-                const char *dir,
-                BootEntry **entries,
-                size_t *n_entries) {
+                const char *dir) {
```

Comment:
assert? (the other ones too)

Reply:
Hmm, what do you mean? I see `assert(config);` added below.

Reply:
yeah, all params are assert()ed on